mod_ci/controllers.py

```diff
@@ -620,6 +621,18 @@ def start_ci():
         return json.dumps({'msg': 'EOL'})
 
 
+def check_status_for_badge(status):
+    if test.test_type != test.pull_request:
+        dwg = svgwrite.Drawing('status.svg', profile='full')
+        if status == Status.SUCCESS:
+            dwg.add(dwg.text('Passing', insert=(0, 0.2), fill='green'))
+        elif status == Status.FAILURE:
+            dwg.add(dwg.text('Failing', insert=(0, 0.2), fill='red'))
+        else:
+            dwg.add(dwg.text('Unknown', insert=(0, 0.2), fill='gray'))
+        dwg.save()  # Couldn't give it an external path it saved it in the local directory...
```

Comment:
That's most likely because you didn't define the path when you created the SVG file ;)

Reply:
when i tested it i gave it external paths and always generated where the script was executed

Reply:
According to the docs, it's possible: http://pythonhosted.org/svgwrite/classes/drawing.html